src/lib/specs-crawler.js

```diff
@@ -97,8 +100,14 @@ async function crawlSpec(spec, crawlOptions) {
             },
             [spec, crawlOptions.modules],
             { quiet: crawlOptions.quiet,
-              forceLocalFetch: crawlOptions.forceLocalFetch }
+              forceLocalFetch: crawlOptions.forceLocalFetch,
+              etag, lastModified}
         );
+        if (result.error === "reuseexistingdata" && fallback) {
```

Comment:
See comment on using `status` instead of `error`:
```suggestion
        if (result.status === "notmodified" && fallback) {
```